server.js

```diff
@@ -11,13 +12,30 @@ const logger = require("morgan");
 const connectDB = require("./config/database");
 const mainRoutes = require("./routes/main");
 const postRoutes = require("./routes/posts");
+const eventsRoutes = require("./routes/events");
 
 //Use .env file in config folder
 require("dotenv").config({ path: "./config/.env" });
 
 // Passport config
 require("./config/passport")(passport);
 
+const whitelist = ['http://localhost:3000']
```

Comment:
@JoeDravarol will there be an issue here? Should this not be port 2121?

Reply:
It should be the port your front-end is running on

Reply:
> @JoeDravarol will there be an issue here? Should this not be port 2121?

No, this is fine. Like @MatthewBozin mentioned it's whitelisting the frontend port so that it can communicate with the backend. At the current stage, It's only used for developing/testing.